pre-commit.sh

```diff
@@ -9,12 +9,14 @@ if ! [ -e venv ]; then
   python3 -m pip install virtualenv
   python3 -m virtualenv -p python3 venv
 fi
-venv/bin/pip install -r requirements-server.txt
-venv/bin/pip install -r requirements.docs.txt
-venv/bin/pip install -r requirements.dev.txt
+venv/bin/pip install -r requirements-server.txt --no-cache
```

Comment:
why did you add --no-cache? isn’t it faster / easier if we don’t have that argument in?

Reply:
I think that without `--no-cache`, you might inadvertently install an older version of the package if it happens to be on your system. Might also need `-U` on these installs, so the venv automatically gets updated if the pins are relaxed..

Reply:
We have version specifications right? Each module is specified to a particular version.